operator/src/main/java/oracle/kubernetes/operator/rest/RestBackendImpl.java

```diff
@@ -68,7 +82,6 @@
    * @param accessToken is the access token of the Kubernetes service account of the client calling
    *     the WebLogic operator REST api.
    * @param targetNamespaces a list of Kubernetes namepaces that contain domains that the WebLogic
-   *     operator manages.
```

Comment:
This is an odd edit to this Javadoc. A mistake?

Reply:
Yes.